### PR TITLE
Pep8 and python2 fixed

### PR DESCRIPTION
**Pull request by sih4sing5hong5**
The environment of tests is ok in [travis](https://travis-ci.org/i3thuan5/fatek-fbs-lib).
There is [coverage](https://coveralls.io/builds/10270402), too.

### CODE REVIEW

File: fatek/target.py [hunk 1]
```diff
@@ -23,14 +23,18 @@ def __assign_functions(self):
...
+        # (start coil, number of readed bits)
+        return self.client.read_coils(self.symbol.offset, 1).bits[0]
 
     def _write_coil(self, value):
         return self.client.write_coil(self.symbol.offset, value)
 
     def _read_holding_r(self):
         # (start , number of readed bytes)
-        return self.client.read_holding_registers(self.symbol.offset, 1).registers[0]
+        return (
```
Comment (netrounds-rimek) [inline fatek/target.py]: no, revert it

File: fatek/__init__.py [hunk 2]
```diff
@@ -1 +0,0 @@
-from .fatek import Fatek
```
Comment (netrounds-rimek) [inline fatek/__init__.py]: I rely on that in my code, which uses this lib

**Comment by netrounds-rimek** [comment]
ok, so maybe merge here stuff you did in the https://github.com/rimek/fatek-fbs-lib/issues/8 and close it, so there will be only one PR

File: fatek/target.py [hunk 3]
```diff
@@ -40,8 +44,16 @@ def read_all(self, count):
         number = self.symbol.offset
         current_value = self.current_value
 
-        if target in ['Y', 'X', 'M', 'S'] or (current_value == False and target in ['T', 'C']):
-            return self.client.read_coils(number, count).bits  # (start coil, number of readed bits)
-
-        elif target in ['R', 'D'] or (current_value == True and target in ['T', 'C']):
-            return self.client.read_holding_registers(number, count).registers  # (start, number of readed bits)
+        if (
```
Comment (netrounds-rimek) [inline fatek/target.py]: no, revert it. If you want to simplify conditions, put they into separate variables. I find it hard to read

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: python
+python:
+- '2.7'
+- '3.4'
+- '3.5'
+install:
+- pip install pymodbus pyserial || pip install pymodbus3 pyserial
```
Comment (netrounds-rimek) [inline .travis.yml]: this is no good, as I commented on the other PR, there should be `install: "pip install -r requirements.txt"` here. And there is a way to set conditions in `requirements.txt` - look at this:
https://www.python.org/dev/peps/pep-0496/#id6

File: fatek/symbol.py [hunk 5]
```diff
@@ -87,9 +88,12 @@ def _verify_type(self):
 
     def _verify_number(self):
         if self.target != 'R':
-            result = self.number in xrange(*self.allowed_numbers[self.target])
+            result = self.number in range(*self.allowed_numbers[self.target])
         else:
-            result = any([self.number in xrange(*numbers) for numbers in self.allowed_r_numbers])
+            result = any(
```
Comment (netrounds-rimek) [inline fatek/symbol.py]: also revert it

File: requirements.txt [hunk 6]
```diff
@@ -1,2 +1,3 @@
 pymodbus3 == 1.0.0
 pyserial == 2.7
+pymodbus
```
Comment (netrounds-rimek) [inline requirements.txt]: according to the https://www.python.org/dev/peps/pep-0496/ you can put conditions here
```
pymodbus3 == 1.0.0; python_version >= '3.0'
pymodbus; python_version <= 3.0
```
and pin the specific version of `pymodbus`

**Review by netrounds-rimek — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 6 above)
Comment (sih4sing5hong5) [reply requirements.txt]: Cool

File: fatek/__init__.py [hunk 7]
```diff
@@ -1 +1,4 @@
 from .fatek import Fatek
+
+
+Fatek
```
Comment (rimek) [inline fatek/__init__.py]: what is that ? :)

**Comment by rimek** [comment]
Looks good, did you check if its working in both python versions ?

File: fatek/tools.py [hunk 8]
```diff
@@ -1,6 +1,5 @@
-from .fatek import Fatek
+# from .fatek import Fatek
 
 
 def fatek_cli(adress, symbol):
-
-    print "#TODO console client"
+    print("#TODO console client")
```
Comment (rimek) [inline fatek/tools.py]: without `from __future__ import print_function` this won't work in python2. You can also leave `pass` here for now

**Review by rimek — CHANGES_REQUESTED** [review]

File: fatek/__init__.py (hunk 7 above)
Comment (sih4sing5hong5) [reply fatek/__init__.py]:
```
$ flake8 . --exclude=venv
./fatek/__init__.py:1:1: F401 '.fatek.Fatek' imported but unused
```
Maybe `__all__` is better than it.

**Comment by sih4sing5hong5** [comment]
python 2/3 worked in my [travis](https://travis-ci.org/i3thuan5/fatek-fbs-lib/builds/204425742).
Could you manipulate code format remained ?

**Pull request merged**